metadata-ingestion-examples/common/build.gradle

```diff
@@ -7,7 +7,6 @@ dependencies {
 
   compile externalDependency.javaxInject
   compile externalDependency.kafkaAvroSerde
-  compile externalDependency.kafkaSerializers
```

Comment:
Ahh it looks like the metadata ingestion examples project got merged around the same time as my PR, sorry about that!

Reply:
No problem. We're also in the process of setting up the slack notification so we'll get notified immediately when the build is broken: https://github.com/linkedin/datahub/pull/1873